LICENSE.md

```diff
@@ -0,0 +1,22 @@
+The MIT License
+
+Copyright (c) 2013-2015 Mashape (https://www.mashape.com)
```

Comment:
I feel like I'm missing something rather obvious, but who is `Mashape`, listed as the copyright holder? Otherwise this looks fine.

Reply:
oh no good catch.  I think I actually copied this licensefile from one of other repo's.... hmmm need to check.